packages/blocks/src/code-block/components/SyntaxCodeBlock.ts

```diff
@@ -0,0 +1,112 @@
+// @ts-nocheck
```

Comment:
I thinks this file should be named as `syntax-code-block.ts` and would be great if we can remove the `@ts-nocheck` in the future.

Reply:
Get that, file should be named this style@ts-nocheck is a trade-off so far, Quill's declartion file doesn't correspond to its packed dist/index.js file, confusing

Reply:
If there are type errors, you should fix the type manually instead ignore all type

Reply:
@lawvs I believe the original file is JavaScript. I can imagine the possible friction. We can do some type-refactoring after the critical features get stable or make it progressive.